.github/workflows/test.yaml

```
@@ -168,3 +168,49 @@ jobs:
       uses: pypa/gh-action-pypi-publish@v1.3.1
       with:
         password: ${{ secrets.PYPI_UPLOAD_TOKEN }}
+
+  lint:
+    runs-on: ubuntu-20.04
+    env:
+      TOX_PARALLEL_NO_SPINNER: 1
+
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        # Need history on all branches for newsfragment to work.
+        fetch-depth: 0
+
+    - name: Set up Python
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.9'
+
+    - name: Install dependencies
+      run: |
+        python -m pip install --upgrade pip tox
+
+    - name: Lint
```

I think the name of the step (and also for the overall job) is no longer `lint` ... but I don't know what name to use :)
maybe "tox run" ?

---

It's stuff that's `not test`

---

What about calling it `nit`?

---

I think that any name, other than 'lint' would be ok :) ... but this is the tox environments run ... 
but nit is a bit gross :p

---

> this is the tox environments run 

the other stages run tox environments - specifically the `py` environments

---

I don't have a better name... but I think that this is a good description for the step

```suggestion
    - name: Run the checks
```